### PR TITLE
VTN-6650 add Sentiment Output Component

### PR DESCRIPTION
**Pull request by KhuongVeritone**

### CODE REVIEW

**Comment by SteveShaffer** [comment]
@KhuongVeritone you'll want to merge `master` into this branch.  Once you do the wercker/build should kick off correctly.

**Comment by SteveShaffer** [comment]
Nevermind.  I got it.  It was easier than GitHub suggested.  Oh wait, I see; we need to also sync with MDP branch....

**Comment by KhuongVeritone** [comment]
yeah, let me try to merge master to MDP

**Comment by SteveShaffer** [comment]
...and then merge MDP branch into this branch potentially

File: packages/veritone-react-common/src/components/SentimentEngineOutput/index.js [hunk 1]
```diff
@@ -1,189 +1,244 @@
...
+  }
 
-  // Map "score" from [0,1] range to [-100,100] range and set "time" to midpoint of start/end:
-  function normalizeDatum(datum) {
-    let normalizedDatum = {
-      score: (datum.score - 0.5) * 200,
-      time: (datum.start + datum.end) / 2
-    };
+  formatXValues (value) {
+    return msToReadableString(value);
```
Comment (KhuongVeritone) [inline packages/veritone-react-common/src/components/SentimentEngineOutput/index.js]: date lib doesn't give us the values that we want (0 hours & days breaks it); therefore, we have to use our custom function to handle time format.

**Pull request merged**